cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -180,7 +180,8 @@ public void getAllApps(HttpRequest request, HttpResponder responder,
         names.add(name);
       }
     }
-    responder.sendJson(HttpResponseStatus.OK, applicationLifecycleService.getApps(namespace, names, artifactVersion));
+    responder.sendJson(HttpResponseStatus.OK, applicationLifecycleService.getApps(namespace.toEntityId(),
```

Comment:
Should I do `NamespaceId namespace = validateNamespace(namespaceId).toEntityId();` or change the return type of `validateNamespace`?